pepper-apis/src/test/resources/db-testscripts/baseline-seed-test-data.xml

```diff
@@ -51,18 +51,17 @@
             </sqlCheck>
         </preConditions>
         <insert tableName="client">
-            <column name="client_name" value="demo-web-client"/>
             <column name="is_revoked" valueBoolean="false"/>
             <column name="auth0_domain" value="https://ddp-dev.auth0.com/"/>
             <column name="auth0_client_id" value="5ps6MnYXmzpuo4OePvfcW0Dfv6cMPNIC"/>
             <column name="auth0_signing_secret" value="20JhCH3DTn905Vm90Rcy/DGNqQw9EJ/XOGvPG8yyQV7+kkL4riGVGKEuTzZ6xKOQMFL9wKA8ZBC8h1WqKI4xeh9dX0BqBfRrAuubSyHPrOP2qUX4DcO4SqXqyO5CEnr4"/>
         </insert>
         <insert tableName="client__umbrella_study">
-            <column name="client_id" valueComputed="(select client_id from client where client_name = 'demo-web-client')"/>
+            <column name="client_id" valueComputed="(select client_id from client where auth0_client_id = '5ps6MnYXmzpuo4OePvfcW0Dfv6cMPNIC' and auth0_domain = 'https://ddp-dev.auth0.com/')"/>
```

Comment:
Hmmm, are we sure we want to put this id in here?

Reply:
Not sure what you mean actually... Since we are not using `client_name` anymore, the only reliable way to find a client is the `auth0_client_id` + `auth0_tenant_id` combination.